### DISCUSSION TITLE
Welcome to Day-0-Hello-World. Discussions!

### DISCUSSION THREAD

**Discussion by mikasaid** [muhammaddzaky/muhammaddzaky#7] [Ideas]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mikasaid** [comment]
Good 
![0](https://user-images.githubusercontent.com/58392246/128862894-54d9d965-cbc1-4910-8996-87169332cc31.gif)
![946514d6322a9a415926d522e2e014e6](https://user-images.githubusercontent.com/58392246/128862906-021ea74c-5aab-4658-aab7-7e552a4a675f.jpg)
![images (1)](https://user-images.githubusercontent.com/58392246/128862907-f32b3953-3473-440d-ad9b-713934bbcd8d.jpg)
![images](https://user-images.githubusercontent.com/58392246/128862909-5e2dd5d2-156f-4e75-8e86-3fa23dd7a109.jpg)

**Comment by dzakysaragih** [comment]
Oke Jugak
